sdk/monitor/monitor-query/package.json

```diff
@@ -69,7 +69,7 @@
     "test:node": "npm run build:test && npm run integration-test:node",
     "test": "npm run build:test && npm run integration-test",
     "unit-test:browser": "echo 'karma start --single-run' temporarily disabled",
-    "unit-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/internal/unit/**/*.spec.ts\"",
+    "unit-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/{,!(browser)/**/}*.spec.ts\"",
```

Comment:
Ok I just looked and it looks like you do have unit tests here: https://github.com/Azure/azure-sdk-for-js/tree/8a133c7b37c0e9ec13c7326f54fadb2d9d25d9d3/sdk/monitor/monitor-query/test/internal/unit and the updated command runs both them and the live tests as can be seen in CI: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1024467&view=logs&j=3cacf272-7c7b-57a7-ca0c-8c0bd3bac579&t=b52a4bca-c821-5c3d-d1d8-be67fee354d7&l=153. So I believe this is a good change.